tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -627,85 +627,85 @@
-    <string name="update_checking_title">Checking Updates</string>
-    <string name="update_checking_message">Please wait while Updates are checking.</string>
+    <string name="update_ipro_has_apdate">Download and install the new version of InviZible Pro in the background?</string>
+    <string name="update_dnscrypt_has_apdate">Download and install the new version of DNSCrypt in the background?</string>
+    <string name="update_tor_has_apdate">Download and install the new version of Tor in the background?</string>
+    <string name="update_itpd_has_apdate">Download and install the new version of Purple I2P in the background?</string>
+    <string name="update_checking_title">Checking for New Versions…</string>
+    <string name="update_checking_message">Please wait while looking for new versions.</string>
     <string name="update_check_warning">The update server is temporarily unavailable. Please try again later.</string>
     <string name="update_check_warning_menu">The update server was unavailable.</string>
     <string name="update_fault">Update fault.</string>
-    <string name="update_check_no_update">No Updates were Found.</string>
+    <string name="update_check_no_update">Everything is up to date.</string>
     <string name="update_last_check">Last check:</string>
     <string name="update_found">Updates were found.</string>
     <string name="update_installed">Updates were installed.</string>
-    <string name="update_over_three_activations">Looks like you have activated 3 copies of InviZible with this code already. If it is wrong, please contact the developer.</string>
-    <string name="update_wrong_code">Looks like your PRO code is wrong. Please contact the developer.</string>
-    <string name="update_over_five_times">You checked InviZible Pro updates more than 5 times a day. Please try again later.</string>
+    <string name="update_over_three_activations">You have activated 3 copies of InviZible with this code already. If it is wrong, please contact the developer.</string>
+    <string name="update_wrong_code">Your Pro code is wrong. Please contact the developer.</string>
+    <string name="update_over_five_times">You checked for new versions of InviZible Pro more than 5 times in a day. Please try again later.</string>
 
-    <string name="verifier_error">Looks like this unofficial version of InviZible. Please be care of it!</string>
-    <string name="only_for_pro">Only for PRO version</string>
+    <string name="verifier_error">This is unofficial version of InviZible. It was not built by the developer.</string>
+    <string name="only_for_pro">Only for the Pro version</string>
     <string name="donate">Donate</string>
-    <string name="donate_project">InviZible Pro Project needs your assistance. Please visit donate page or enter received premium code.</string>
+    <string name="donate_project">The InviZible Project needs your help. Please visit the donate page or enter your received premium code.</string>
```

Comment:
```suggestion
    <string name="donate_project">The InviZible Project needs your help. Please visit the donate page or enter your received Pro code.</string>
```
I like "premium" better, but are those the same thing? "Pro" is used more often.